common/cmake/gen.cmake

```diff
@@ -115,15 +114,85 @@ function(V2X)
       ${INCLUDE_ARG}
     WORKING_DIRECTORY ${symbiflow-arch-defs_SOURCE_DIR}/utils/vlog/
   )
-
   add_file_target(FILE "${V2X_NAME}.model.xml" GENERATED)
   get_file_target(SRC_TARGET_NAME "${V2X_NAME}.model.xml")
   set_target_properties(${SRC_TARGET_NAME} PROPERTIES INCLUDE_FILES "${MODEL_INCLUDE_FILES}")
 
+  add_custom_command(
```

Comment:
This belongs in a new function, not as part of the V2X function itself.  For example, this could be used on an non-top-level pb_type.

Reply:
Done.